launchdarkly-server-sdk/src/events/sender.rs

```diff
@@ -134,6 +134,7 @@ where
 
                 let response = match result {
                     Ok(response) => response,
+                    Err(_) if attempt == 1 => continue,
```

Comment:
A customer opened an issue because their logs are filled with the message show in the other error branch. While this change doesn't necessarily fix their specific problem, it should help in a couple of ways.

1. Originally, we were considering the original failure to be unrecoverable, which might not be true. So more events might start making it through.
2. We won't log anything on the first failure, so if the failure is temporary, we won't unnecessarily alert customers.